### ISSUE TITLE
Name and role can be programmatically determined for every UI component

### ISSUE THREAD

**Issue by Dananji** [avalonmediasystem/structured-metadata-editor-react#77]
### Description
SME audit report line item 4.1.2, referenced in this issue:  #64 

WCAG 2.1 Checkpoint for Name, Role, Value  - Control elements within the audio component do not have labels to distinguish roles of each element.

The default controls that comes with the HTML audio tag does not have descriptive labels for each button, seek bar, volume control in it. The Chrome VOX screen reader was not able to identify each of this, when page was navigated through keyboard.

### Done Looks Like
- Implement custom audio controls

### To Do
- [ ] Implement a solution
- [ ] Re-test

### Notes
Suggested solution for accessible multimedia: https://developer.mozilla.org/en-US/docs/Learn/Accessibility/Multimedia
